src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,41 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private CinemaHallService cinemaHallService;
+    private CinemaHallDtoMapper mapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService, CinemaHallDtoMapper mapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.mapper = mapper;
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAll() {
+        return cinemaHallService.getAll()
+                .stream()
+                .map(mapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    public CinemaHallResponseDto create(@RequestBody CinemaHallRequestDto request) {
```

Comment:
```suggestion
    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto request) {
```
the same for other create methods, need to save consistency in method names. If you use add on service layer, use it on controllers layer too

Reply:
look more attentively at this comment